freetextresponse/public/view.js

```diff
@@ -102,6 +97,16 @@ function FreeTextResponseView(runtime, element) {
         return html;
     }
 
+    function display_responses_if_answered(response) {
+        if (!response.user_alert && response.display_other_responses) {
+            var responseHTML = get_student_responses_html(response.other_responses);
```

Comment:
And while we're in here, let's explicitly hoist the variable to the top of the function and update our naming to match JS convention (camelCase instead of snake_case).

`responseHtml` and `getStudentResponsesHtml`.